### DISCUSSION TITLE
Next Release?

### DISCUSSION THREAD

**Discussion by chmorgan** [chmorgan/sharppcap#200] [General]
Hi @kayoub5. If you think the send packet changes are stable enough I'd like to release 5.4 so we can move on to 6.0 and get those api breaking changes separated out.

**Comment by kayoub5** [comment]
@chmorgan I would say they are stable enough, just to be safe, you may want to create a branch from the 5.4 release, in case someone request/propose a hot-fix for some unrelated topic.

**Comment by kayoub5** [comment]
@chmorgan I don't have any new major features planned soon, and current master is stable enough.

Could you publish a new version?

**Reply by chmorgan** [reply]
Sounds good. I'm seeing emails about an odd nuget issue:

```
The symbol package SharpPcap 6.1.0 failed validation because of the following reason(s):

The uploaded symbols package contains pdb(s) for a corresponding dll(s) not found in the nuget package.
Your symbol package was not published on NuGet Gallery and is not available for consumption.

You can reupload your symbol package once you've fixed the issue with it.
```

**Reply by kayoub5** [reply]
that would be https://github.com/NuGet/Home/issues/9647

**Comment by kayoub5** [comment]
@chmorgan could you publish a new version? we have pending changes in master that were never released.

**Comment by chmorgan** [comment]
Hmm. @kayoub5 it should push automatically through the CI system. Did we bump up the various versions that are checked during the nuget push? I think if those don't change the new packages aren't accepted.

**Reply by kayoub5** [reply]
It seems the version was not bumped after the latest changes to master, I just made a commit to bump the version.
